fe/fe-core/src/main/antlr4/org/apache/doris/nereids/DorisParser.g4

```diff
@@ -717,6 +722,21 @@ primaryExpression
     | primaryExpression COLLATE (identifier | STRING_LITERAL | DEFAULT)                        #collate
     ;
 
+partitionFunctionCallExpression
+    : LEFT_PAREN partitionFunctionList RIGHT_PAREN
+    ;
+
+partitionFunctionList
+    : functions+=partitionFunction (COMMA functions+=partitionFunction)*
+    ;
+
+partitionFunction
```

Comment:
it is better use `functionCallExpression` directly, and in LogicalPlanBuilder check return expression is `UnboundFunction`